### DISCUSSION TITLE
@babel/runtime dependency necessary

### DISCUSSION THREAD

**Discussion by p-kuen** [remirror/remirror#2130] [General] [answered]
Hello,

I found no issue or PR regarding this question:
What is the reason that almost every package in this repo has the @babel/runtime dependency listed in package.json?
For example in the prosemirror-trailing-node package I found no reference to the babel runtime. Can it be removed?

Thanks!

Regards.

**Comment by ocavue** [comment, marked as answer]
Yes. Most `@babel/runtime` can be removed. Remirror used to use `babel` to transpile the source code, and that's why we have `@babel/runtime`. However, most Remirror packages are using `esbuild` now so they can be removed. I will do that if I have some time (PRs are also welcome).

**Reply by p-kuen** [reply]
Thanks for the quick response! I will try to add PRs for some packages.
